Makefile

```diff
@@ -105,7 +105,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-build: godeps-update test-setup ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	podman build -t ${IMG} .
```

Comment:
Can you revert this change

Reply:
oops. this shouldn't have been part of the PR. changing.